### PR TITLE
Update images for guestbook sample app

### LINKED ISSUE #914: "Existing application" tutorial failed since the container tag was removed from repository.
The [existing application sample](https://github.com/radius-project/samples/blob/v0.28/samples/kubernetes/guestbook/deploy/frontend-deployment.yaml#L19) described in https://docs.radapp.io/tutorials/add-radius/
 refers `gcr.io/google-samples/gb-frontend:v4` image.

However, the tag was removed from the repository.

https://console.cloud.google.com/gcr/images/google-samples/GLOBAL/gb-frontend

A similar issue has been reported in kubernetes docs. https://github.com/kubernetes/website/issues/44224

I also tried to specify the sha256 tag like `gcr.io/google-samples/gb-frontend@sha256:e5233a1e2a6dc9c4ae1d1b754e3a3f62ecdc1da81b31422aff4e7390a0c6e534`, the deployed application failed since it can't connect to redis container.

[AB#10810](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/10810)

### PR DESCRIPTION
**Pull request by willtsai**
This is to address https://github.com/radius-project/samples/issues/914 where a broken dependency on an image in the Google container registry is broken. 

We fix the issue by downloading and pushing a copy of the required images into the Radius container registry so that we no longer have a dependency on Google's container registry:

- gcr.io/google_samples/gb-redisslave:v1 --> ghcr.io/radius-project/gb-redisreplica:v1 
- gcr.io/google-samples/gb-frontend:v4 --> ghcr.io/radius-project/gb-frontend:v4  

Fixes: https://github.com/radius-project/samples/issues/914

### CODE REVIEW

**Comment by willtsai** [comment]
confirmed that the fix addresses the issue:

```bash
> kubectl get deployments -n demo                  
NAME            READY   UP-TO-DATE   AVAILABLE   AGE
redis-master    1/1     1            1           75m
redis-replica   2/2     2            2           75m
frontend        3/3     3            3           75m
> kubectl get services -n demo
NAME            TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)        AGE
frontend        NodePort    10.43.51.86   <none>        80:30096/TCP   75m
redis-master    ClusterIP   10.43.49.67   <none>        6379/TCP       75m
redis-replica   ClusterIP   10.43.6.205   <none>        6379/TCP       75m
```

**Comment by willtsai** [comment]
Since we don't expect these images to ever need any changes, I simply pulled a copy from their original (Google) registries and pushed the copies to the Radius registry:
- https://github.com/orgs/radius-project/packages/container/package/samples/gb-frontend
- https://github.com/orgs/radius-project/packages/container/package/samples/gb-redisreplica

Thus, I did not set up pipelines for these images as I don't think we need to given they should not need any changes as a sample.

File: samples/kubernetes/guestbook/deploy/frontend-deployment.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
       - name: php-redis
-        image: gcr.io/google-samples/gb-frontend:v4
+        image: ghcr.io/radius-project/gb-frontend:v4
```
Comment (Reshrahim) [inline samples/kubernetes/guestbook/deploy/frontend-deployment.yaml]: should we follow the same registry path as demo and eshop samples `ghcr.io/radius-project/samples/<sampleapp>`?

Comment (willtsai) [reply samples/kubernetes/guestbook/deploy/frontend-deployment.yaml]: good catch! updated.

**Review by Reshrahim — APPROVED** [review]

**Pull request merged**